tests/ember-configuration.js

```diff
@@ -119,11 +122,11 @@
     };
 
     // Prevent all tests involving serialization to require a container
-    DS.JSONSerializer.reopen({
-      transformFor: function(attributeType) {
-        return this._super(attributeType, true) || transforms[attributeType];
-      }
-    });
+    var transformFor = function(attributeType) {
+      return this._super(attributeType, true) || transforms[attributeType];
+    };
+    DS.JSONSerializer.reopen({ transformFor: transformFor });
+    DS.JSONAPISerializer.reopen({ transformFor: transformFor });
```

Comment:
Looks like `transformFor` is never defined on the serializer and adapter, but the tests pass because of this patching (would not work in an actual app). I think?